packages/file/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+#### 5.1.0
+
+* Added `clock` argument to `MemoryFileSystem`
```

Comment:
technically it just adds another constructor that forces the clock to a test clock